### ISSUE TITLE
Google indexing ? 

### ISSUE THREAD

**Issue by Audeh007** [905Palestine/905Palestine.github.io#3]
I am not seeing the website pop up on google searches despite searching for “ Niagara Palestine Coalition” 
Did we submit a request for indexing with google?  Do we have a site map?

**Comment by 905Palestine** [comment]
Shouldn't need a sitemap since it's just one page but it looks like I had mistakenly used an unmasked redirect from 905palestine.ca; [after I changed it to a 301 it showed up when typing the full url](https://www.google.com/search?q=About+https://niagarapalestinecoalition.ca/&tbm=ilp&sa=X&ved=2ahUKEwi3_Z7JwY6GAxUQL2IAHQWhAYsQv5AHegQIABAC). It'll take a few days for google to fully play nice. I'll also put our website on more of our socials, that usually helps SEO. I also registered with google search console so I'll update the issue when I get more info from that.
